### DISCUSSION TITLE
Hatchery, Load and Pause

### DISCUSSION THREAD

**Discussion by registant** [mindryder/ATLAS-RU#3] [НАЙДЕННЫЙ ПРОПУЩЕННЫЙ ПЕРЕВОД]
![Screenshot_1322](https://user-images.githubusercontent.com/123338303/214041374-221a8b13-9953-47ad-8d11-4b776ff77cd7.jpg)
![Screenshot_1323](https://user-images.githubusercontent.com/123338303/214041651-c0f3e3ec-e90f-4e91-919b-36d8de92327c.jpg)
![Screenshot_1324](https://user-images.githubusercontent.com/123338303/214042364-f1eff242-1953-465c-acf7-a7d14e21f76c.jpg)

**Comment by mindryder** [comment]
Для Tame Hatchery - разработчики не выложили переменной пока невозможно
Load into - переведено, добавлено в новый релиз
Pause  - разработчики не выложили переменной пока невозможно

Спасибо за отслежку пропущенного

**Comment by registant** [comment]
![Screenshot_1337](https://user-images.githubusercontent.com/123338303/214249715-4af52729-69f2-4edc-93b5-4fa760cce199.jpg)
Вот еще это

**Reply by registant** [reply]
Обновил до 1.3

**Reply by mindryder** [reply]
Load Nearby - разработчики не выложили переменной пока невозможно

**Comment by registant** [comment]
![Screenshot_1372](https://user-images.githubusercontent.com/123338303/215037420-f9a7c6ec-212d-49bb-883d-9cde00b1a16a.jpg)

**Reply by mindryder** [reply]
Спасибо. Переведено.
